File: tests/expected/ifexp.cpp
#include <algorithm>  // NOLINT(build/include_order)
#include <cstdint>    // NOLINT(build/include_order)
#include <iostream>   // NOLINT(build/include_order)
#include <vector>     // NOLINT(build/include_order)
inline void show() {
  int a = 1;
  auto b = (({
    std::vector<int> __tmp1 = {2, 3};
    (std::find(__tmp1.begin(), __tmp1.end(), a) != __tmp1.end());
  })
                ? ({ 2; })
                : ({ 3; }));
  std::cout << b;
  std::cout << std::endl;
}

int main(int argc, char** argv) { show(); }

File: tests/expected/ifexp.dart
// @dart=3.4
import 'package:sprintf/sprintf.dart';

show() {
  final int a = 1;
  final b = [2, 3].contains(a) ? (2) : (3);
  print(sprintf("%s", [b]));
}

main(List<String> argv) {
  show();
}
